src/test/app/AMM_test.cpp

```diff
@@ -4858,25 +4858,18 @@ struct AMM_test : public jtx::AMMTest
         using namespace jtx;
         FeatureBitset const all{testable_amendments()};
         FeatureBitset const noAMM{all - featureAMM};
-        FeatureBitset const noNumber{all - fixUniversalNumber};
-        FeatureBitset const noAMMAndNumber{
-            all - featureAMM - fixUniversalNumber};
 
-        for (auto const& feature : {noAMM, noNumber, noAMMAndNumber})
-        {
-            Env env{*this, feature};
-            fund(env, gw, {alice}, {USD(1'000)}, Fund::All);
-            AMM amm(env, alice, XRP(1'000), USD(1'000), ter(temDISABLED));
-
-            env(amm.bid({.bidMax = 1000}), ter(temMALFORMED));
-            env(amm.bid({}), ter(temDISABLED));
-            amm.vote(VoteArg{.tfee = 100, .err = ter(temDISABLED)});
-            amm.withdraw(WithdrawArg{.tokens = 100, .err = ter(temMALFORMED)});
-            amm.withdraw(WithdrawArg{.err = ter(temDISABLED)});
-            amm.deposit(
-                DepositArg{.asset1In = USD(100), .err = ter(temDISABLED)});
-            amm.ammDelete(alice, ter(temDISABLED));
-        }
+        Env env{*this, noAMM};
```

Comment:
```suggestion
        Env env{*this, testable_amendments() - featureAMM};
```

This pattern is widely used throughout the codebase.